robot-core/src/main/java/org/obolibrary/robot/TemplateHelper.java

```diff
@@ -991,6 +991,14 @@ protected static OWLClassExpression tryParse(
         // If we have a checker, try to get the class by label
         // This allows class labels with single quotes
         expr = checker.getOWLClass(content, false);
+        if (expr == null) {
+          // If not found by label, is it just an IRI?
+          IRI iri = IRI.create(content);
```

Comment:
@jamesaoverton if someone enters a CURIE using an unknown prefix, is there a danger of turning it into an IRI here? In my experience OWL API will turn those into fake IRIs where the prefix becomes a scheme, e.g., "UNKNOWN:1234" -> `<UNKNOWN:1234>`. I find those mistakes really maddening (just speaking in general, not about robot template). Should you make some assumptions about IRIs here, such as only doing this if it starts with `http:`, `https:`, `urn:`, etc.?

Reply:
Good point. I agree.

Reply:
I added a unit test that 'UNKNOWN:1234' fails, and it did without any other changes. But I also added a check for HTTP IRIs.